db/db.js

```diff
@@ -0,0 +1,61 @@
+import fastifyPlugin from "fastify-plugin";
+import { drizzle as postgresDrizzle } from "drizzle-orm/postgres-js";
+import { migrate as postgresMigrate } from "drizzle-orm/postgres-js/migrator";
+import { drizzle as sqliteDrizzle } from "drizzle-orm/better-sqlite3";
+import { migrate as sqliteMigrate } from "drizzle-orm/better-sqlite3/migrator";
+import postgres from "postgres";
+import Database from "better-sqlite3";
+import config from "../config.js";
+
+// PostgreSQL schema
+import * as postgresSchema from "./postgres/schema.js";
+// SQLite schema
+import * as sqliteSchema from "./sqlite/schema.js";
+
+const dbPlugin = async function (fastify) {
+  let db, users, authenticator, storage;
+
+  try {
+    if (config.POSTGRESQL_ENABLED) {
```

Comment:
can we try parsing the connection string ie: look for sqlite or postgresql in the string and automatically enable the right DB?